cookie-banner-rules-list.json

```diff
@@ -2855,9 +2789,9 @@
     },
     {
       "click": {
-        "optIn": "button[data-testid=\"cookie-banner-strict-accept-all\"]",
-        "optOut": "button[data-testid=\"cookie-banner-strict-accept-selected\"]",
-        "presence": "div[data-testid=\"dl-cookieBanner\"]"
+        "optIn": ".text-deepl-blue active:bg-dark-7 rounded-xs flex items-center justify-center bg-white px-4 py-1 font-medium",
+        "optOut": ".active:border-dark-7 active:text-dark-7 rounded-xs flex items-center justify-center border border-white px-4 py-1 font-medium text-current",
+        "presence": ".focus-visible-light-container bg-deepl-blue text-white sm:rounded-lg"
```

Comment:
These selectors rely on styling classes rather than stable identifiers that are cookie banner specific. Are the old selectors not working anymore? Looking at the sites DOM the attributes they use are still there. Also `document.querySelector` still returns the buttons properly. Please double check!